edb/server/server.py

```diff
@@ -936,6 +936,10 @@ def reload_tls(_file_modified, _event, retry=0):
         if client_ca_file is not None:
             self.monitor_fs(client_ca_file, reload_tls)
 
+    def start_watching_files(self):
+        # TODO(fantix): include the monitor_fs() lines above
```

Comment:
What is this TODO and why isn't it done in this PR?

Reply:
Ah that was for the watches in `init_tls()` above - it has some complications and it was too late last night. I'll fix it today with either this PR or a new one.